flocx_market/db/sqlalchemy/models.py

```diff
@@ -35,7 +37,8 @@ class Bid(Base):
     start_time = orm.Column(orm.DateTime(timezone=True), nullable=False)
     end_time = orm.Column(orm.DateTime(timezone=True), nullable=False)
     duration = orm.Column(orm.Integer, nullable=False)
-    status = orm.Column(orm.String(15), nullable=False, default='available')
+    status = orm.Column(
+        orm.String(15), nullable=False, default=statuses.AVAILABLE)
```

Comment:
As long as we're using status constants, why not make this an integer (or a single character) instead of a 15-character string?

Reply:
I do think there's some value to making the status readable when querying a database.